src/ocean/core/array/Transformation.d

```diff
@@ -666,3 +666,50 @@ unittest
     int x;
     int[] arr = toArray(x);
 }
+
+/*******************************************************************************
+
+    Assigns elements to an array.
+
+    The function does not allocate GC memory and ensures the length of the
+    array matches the number of variadic arguments.
+
+    Params:
+        T = the type of array element
+        array = the array to assign the elements to
+        elements = the elements to add to the array
+
+*******************************************************************************/
```

Comment:
It should be mentioned in the docs that this is a replacement for declaring an array literal.

Reply:
Right, I'll add that information in the documentation once this PR is code-wise approved.